.travis.yml

```diff
@@ -1,15 +1,37 @@
 os: linux
 dist: xenial
 language: node_js
-node_js:
-  - 10
-  - 12
 
 jobs:
   include:
-    - name: "Lint code and commit message format"
-      node_js: "10"
-      env: VALIDATE_COMMIT_MSG=true LINT=true
+    - node_js: "10"
+      script:
+        - npm run init
+        - npm run build
+        - npm run test:unit
+        - npm run test:e2e
+
+    - node_js: "12"
+      script:
+        - npm run init
+        - echo "TRAVIS_COMMIT $TRAVIS_COMMIT"
+        - echo "TRAVIS_PULL_REQUEST_SHA $TRAVIS_PULL_REQUEST_SHA"
+        - COMMIT_TO_VALIDATE=${TRAVIS_PULL_REQUEST_SHA:-$TRAVIS_COMMIT}
+        - echo "COMMIT_TO_VALIDATE $COMMIT_TO_VALIDATE"
+        - ./scripts/validate-commit-msg.sh $COMMIT_TO_VALIDATE
+        - npm run lint
+        - npm run build
```

Comment:
Should we run this job first, so commit and lint errors are first?

Reply:
Both jobs are run simultaneously, so order does not really matter here.